kong/plugins/opentelemetry/handler.lua

```diff
@@ -28,7 +29,7 @@ local _log_prefix = "[otel] "
 
 local OpenTelemetryHandler = {
   VERSION = "0.1.0",
-  PRIORITY = 14,
+  PRIORITY = 99999,
```

Comment:
how "breaking" is this?

Reply:
This is _potentially_ a breaking change. We've been very careful when changing priorities as this can mess with user expectations, plugin execution chains etc..

Reply:
reverting this for now (more info in design document)